juniper_official/interface/check-optical-signal-loss-fec-tx-rx-power.rule

```diff
@@ -0,0 +1,420 @@
+/*
+ * Detects interface optics signal loss,checks for fec uncorrected errors and monitors
```

Comment:
Rephrase comment something like Monitor interface optics stats like signal loss, power, and errors and notify anomalies